pages/forms/homepage/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" >
+    <meta property="og:title" content="We Life Post" />
+    <meta property="og:type" content="page.png" />
+    <meta property="og:url" content="" />
```

Comment:
Empty url?

Reply:
Yeah, I don't have the vercel url yet so I can't use it. Since I have it I put it on. I think that by mixing it the notification will appear. but I'm not sure about that.